operator/deploy/olm-catalog/0.1.0/automationbroker.crd.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: automationbrokers.automationbroker.io
+  name: automationbrokers.osb.openshift.io
```

Comment:
I had been using `operator/deploy/crds/osb_v1alpha1_ansibleservicebroker_crd.yaml` to test this out in the beginning. The CRDs in the olm-catalog (or manifests...wherever these end up living in the end) could probably be just copies from the `operator/deploy/crds` dir. Thoughts on this and the `AutomationBroker` kind?

https://github.com/openshift/ansible-service-broker/blob/master/operator/deploy/crds/osb_v1alpha1_ansibleservicebroker_crd.yaml

Reply:
I hadn't considered `ansible-service-broker` being an instance of `AutomationBroker`. Reading back through the PR this language makes sense. So I agree with both:

1. Get rid of the `operator/deploy/crds` directory for now to avoid confusion letting the manifest directory be the source of truth
1. Use `AutomationBroker`